docker-compose.yaml

```diff
@@ -30,6 +30,10 @@ services:
     container_name: sawtooth-settings-tp-local
     depends_on:
       - validator
+    environment:
```

Comment:
With these being passed as in as environment variables, can we remove them from the `args:` section above?

Reply:
No, they serve different purpose as explained earlier.